### PR TITLE
docs(@cubejs-backend/docker): improve `README.md`

### PR DESCRIPTION
**Pull request by hassankhan**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Improved the Docker README with some more instructions

### CODE REVIEW

File: packages/cubejs-docker/README.md [hunk 1]
```diff
@@ -1,25 +1,100 @@
...
+- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) -
+  Latest stable release, based on Debian (recommended)
+- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) -
+  Latest stable release on Alpine
+- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) -
+  Latest development release from master branch.
+
+# Quick reference
+
+Where to get help? Check out our Slack, Stack Overflow or GitHub Discussions
```
Comment (rpaik) [inline packages/cubejs-docker/README.md:18]: Should there be links for our Slack, GH Discussions/Issues?

File: packages/cubejs-docker/README.md [hunk 2]
```diff
@@ -1,25 +1,100 @@
...
+pages.
+
+Found an issue? File it on our GitHub Issues page.
+
+Supported architectures: `amd64`
```
Comment (rpaik) [inline packages/cubejs-docker/README.md:23]: Only AMD64?

File: packages/cubejs-docker/README.md (hunk 1 above)
Comment (hassankhan) [reply packages/cubejs-docker/README.md:18]: Whoops 😅 great catch @rpaik 👍

File: packages/cubejs-docker/README.md (hunk 2 above)
Comment (hassankhan) [reply packages/cubejs-docker/README.md:23]: I think as of today, yes. 

@cube-js/core can you confirm please?

Comment (ovr) [reply packages/cubejs-docker/README.md:23]: Yes, probably We will support ARM64 in the feature - https://github.com/cube-js/cube.js/pull/1388

File: packages/cubejs-docker/README.md [hunk 3]
```diff
@@ -1,25 +1,105 @@
...
 
 # How to use this image
 
-```sh
+## Start a Cube.js instance
+
+```bash
 docker pull cubejs/cube:latest
+docker run -d -p 3000:3000 -p 4000:4000 \
+  -e CUBEJS_DB_HOST=postgres://username:password@hostname:port/dbname \
```
Comment (keydunov) [inline packages/cubejs-docker/README.md:58]: It seems CUBEJS_DB_HOST contains not only the host in this example but a full DB URL string.

**Comment by keydunov** [comment]
I'm not sure we need to recommend using `latest`

**Comment by igorlukanin** [comment]
> latest - Latest stable release based on Debian (recommended)

On Docker Hub, we recommend to use the `latest` release however I'm not sure how it's enforced that this release is always stable.

**Comment by keydunov** [comment]
@hassankhan Maybe remove the recommended piece and change the supported tags section to -

**Comment by hassankhan** [comment]
@keydunov Totally agree, updated the PR

**Comment by keydunov** [comment]
Looks good, let's merge it.

**Pull request merged**